views/view.jade

```diff
@@ -5,233 +5,255 @@ block body
+                    li
+                      a(class="fa fa-question", title="Network view tour")
+                    li
+                      div(class="dropdown")
+                        a(class="fa fa-download dropdown-toggle", id="networkDownload", data-toggle="dropdown", aria-haspopup="true", aria-expanded="false")
+                        ul(class="dropdown-menu", aria-labelledby="networkDownload")
+                          li
+                            a(class="save-figure-link", data-format="pdf", data-selector="network", title="Download network view as PDF") PDF
+                          li
+                            a(class="save-figure-link", data-format="png", data-selector="network", title="Download network view as PNG") PNG
+                          li
+                            a(class="save-figure-link", data-format="svg", data-selector="network", title="Download network view as SVG") SVG
+                    li
+                      a(class="print-figure-link fa fa-print", data-selector="network", title="Print network view", id="printSubNet")
+                    li
+                      a(class="fa fa-caret-square-o-down collapse", title="Toggle network view", data-toggle="collapse", data-target="#collapseNetwork", id="networkLink")
 
-      div(id="networkTranscriptRow")
-        div(class="half-page-view leftside")
-            h3
-              | Network
-              small(style="font-size:12px")
-                | &nbsp;&nbsp;
-                a(id="printSubNet", title="Print network.")
-                    span(class="glyphicon glyphicon-print")
-            div(id="network")
+                div(id="collapseNetwork", class="panel-collapse collapse in")
+                  div(id="network", class="panel-body")
+            
+            //- Transcript view
+            div(class="col-lg-6")
+              div(class="panel panel-default")
+                div(class="panel-heading clearfix")
+                  h4(class="panel-title pull-left")
+                    | Transcript
+                  ul(class="list-inline pull-right")
+                    li
+                      a(class="fa fa-info", title="Transcript view description", tabindex="0", data-toggle="popover", data-trigger="hover", data-html="true", data-placement="bottom", data-content="Shows the mutations in each REFSEQ transcript of each gene in the subnetwork. Mutations are colored by cancer type. Each symbol represents a distinct mutation type. Putatively inactivating mutations are shown below the transcript, and the remaining mutations are shown above the transcript. Select one of the three protein domain databases, including Pfam, SMART, and NCBI Conserve Domain, to show annotated protein domains for each transcript -- scroll over a domain to reveal its name. You can <b>zoom in on and drag the transcript annotation</b> by scrolling and/or clicking the figure with your mouse.")
```

Comment:
The description refers selectors for protein domain databases (Pfam, SMART, etc.).  Are they visible somewhere?